website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -117,3 +120,108 @@ If you aren't receiving the invitation email, please check the value for ```APPS
 
 However, if you encounter any issues, you can contact the support team on [Discord](https://discord.com/invite/rBTTVJp) or ask questions on the [community forum](https://community.appsmith.com).
+
+## Server mismatch error
+
+If your server keeps shutting down and displays a schema mismatch error, it may be due to a missed checkpoint upgrade. 
+
+#### Error message
+
+```bash
+Looks like you skipped some required update(s), please go back to the mandatory upgrade path v1.9, or refer to ''https://docs.appsmith.com/'' for more info
+```
+
+#### Cause
+As the error message suggests that your Appsmith instance has missed a required checkpoint upgrade. This could have occurred if you are manually upgrading and have skipped a version, or if you are on automatic upgrade but your instance hasn't checked for updates for a certain period of time.
+
+#### Solution
+
+To fix this error, you may need to manually set your instance to the required version, `v1.9`. To do this, access your Appsmith installation instance and make the following change to the `docker-compose.yml` file:
+
+<Tabs groupId="operating-systems">
+  <TabItem value="CE" label="Community Edition" default>
+  
+    From:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce
+    container_name: appsmith
+```
+
+To:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce:v1.9.0
+    container_name: appsmith
+```
+
+Save this file, then run the following command on your shell to restart Appsmith.
+</TabItem>
+
+   <TabItem value="BE" label="Business Edition">
+   
+From:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ee
+    container_name: appsmith
+```
+
+To:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ee:v1.9.0
+    container_name: appsmith
+```
+
+Save this file, then run the following command on your shell to restart Appsmith.
+   
+   
+   </TabItem>
+</Tabs>
+
+```
+docker-compose up -d
+```
+
+With this your server starts with the older version that includes all necessary schema changes for your Appsmith instance. Once this process has completed successfully (you can verify this by attempting to log in to your Appsmith instance), you can return to the original version of the `docker-compose.yml` file and resume normal operations.
+
+
+<Tabs groupId="operating-systems">
+  <TabItem value="CE" label="Community Edition" default>
+
+  ```bash
+  services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce
+    container_name: appsmith
+  ```
+  
+  </TabItem>
+  <TabItem value="BE" label="Business Edition">
+
+  ```
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ee
+    container_name: appsmith
+  ```
+  </TabItem>
+</Tabs>
+
+
+Now, run the restart command again: 
+
+ ```
+ docker-compose up -d
+  ```
+
```

Comment:
Please add the note shared by Nidhi here. 

```suggestion
:::note If you have not been using your Appsmith instance for a while, it could be possible that there might be multiple versions that show up in the error message. In such a case, perform the same steps for each version in order, starting from the oldest.
:::
```